llama_stack/providers/remote/inference/fireworks/fireworks.py

```diff
@@ -234,6 +237,14 @@ async def _nonstream_chat_completion(self, request: ChatCompletionRequest) -> Ch
 
     async def _stream_chat_completion(self, request: ChatCompletionRequest) -> AsyncGenerator:
         params = await self._get_params(request)
+        yield ChatCompletionResponseStreamChunk(
+            event=ChatCompletionResponseEvent(
+                delta=TextDelta(text=""),
+                event_type=ChatCompletionResponseEventType.prepare,
+                input_prompt=params.get("prompt", None),
+                input_messages=params.get("messages", None),
```

Comment:
Wondering why do we want to return the intermediate response sent to the provider back to the users? I'm a little worried that the returned response would get overloaded with server debug info, should we return a request_id instead that can be used to query telemetry with?